python/golem_task_api/client.py

```diff
@@ -63,10 +66,15 @@
         try:
             response = await client.Check(request)
             if response.status == HealthCheckResponse.SERVING:
+                logger.debug('Service health check completed with success')
                 return channel
         except (StreamTerminatedError, ConnectionError):
             pass
         channel.close()
+        logger.debug(
+            'Service health check failed, will try again in 100ms. deadline=%r',
```

Comment:
Maybe use a common variable for the delay - (100ms and asyncio.sleep(0.1))

Reply:
good one! added the var, did print a float to make it more the same